svc-api/handle/telemetry.go

```diff
@@ -303,8 +304,30 @@ func (a *TelemetryRPCs) GetTrigger(ctx iris.Context) {
 
 // UpdateTrigger is the handler for getting TelemetryService details
 func (a *TelemetryRPCs) UpdateTrigger(ctx iris.Context) {
+	var reqBody interface{}
+
+	err := ctx.ReadJSON(&reqBody)
+	if err != nil {
+		errorMessage := "error while trying to get JSON body from the account update request body: " + err.Error()
+		log.Error(errorMessage)
+		response := common.GeneralError(http.StatusBadRequest, response.MalformedJSON, errorMessage, nil, nil)
+		ctx.StatusCode(http.StatusBadRequest) // TODO: add error headers
+		ctx.JSON(&response.Body)
+		return
+	}
+	request, err := json.Marshal(reqBody)
```

Comment:
Please replace ctx.ReadJSON and json.Marshal with single call to ctx.GetBody() ([]byte, error) to get the payload in byte array.

Reply:
Done